### DISCUSSION TITLE
Is it possible to duplicate an existing Supabase project, and keep all the configuration identical?

### DISCUSSION THREAD

**Discussion by lumenwrites** [supabase/supabase#10567] [Questions]
I'm building a website, and sometimes I want to be able to experiment with various Supabase configurations, database schema, and other stuff, without losing the previous versions (in case I need to go back to them).

Recreating Supabase project from scratchevery time is very error prone and time consuming - it takes me like 1-2 hours to set everything up (the RLS policies, auth config, trigger functions, hooks, new API keys, etc, etc).

Is there a way for me to duplicate an existing project, and keep all of the things I have mentioned identical between the two copies?

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects
The basic migration will cover RLS, tables, data, functions.  But not setup of auth, storage, etc.
